test/support/helpers/review-charge-reference.helper.js

```diff
@@ -54,9 +54,9 @@ function defaults (data = {}) {
     canalAndRiverTrustAgreement: false,
     twoPartTariffAgreement: true,
     winterDiscount: false,
-    abatementAgreement: 0,
-    chargeAdjustment: 0,
-    amendedChargeAdjustment: 0,
+    abatementAgreement: 1,
```

Comment:
The comment text at the top of the file needs to be updated for the change to `abatementAgreement`, this currently says 0.

Reply:
Oops